### PR TITLE
Request allow list (EXPOSUREAPP-10570)

### PR DESCRIPTION
**Pull request by mtwalli**
## Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10570

### Subtask for EXPOSUREAPP-10197

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/allowlist/repo/storage/DccTicketingAllowListStorage.kt [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import de.rki.coronawarnapp.util.serialization.fromJson
+import kotlinx.coroutines.sync.Mutex
+import kotlinx.coroutines.sync.withLock
+import timber.log.Timber
+import javax.inject.Inject
+
+@Reusable
+class DccTicketingAllowListStorage @Inject constructor(
+    @AppContext context: Context,
+    @BaseGson private val gson: Gson
```
[the remark this answers is not in the document]
Comment (BMItr) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/allowlist/repo/storage/DccTicketingAllowListStorage.kt]: done

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/allowlist/data/DccTicketingValidationServiceAllowListEntry.kt [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+@Parcelize
+data class DccTicketingValidationServiceAllowListEntry(
+    /** A display name for the provider of the Validation Service */
+    @SerializedName("serviceProvider") val serviceProvider: String,
+
+    /** The hostname of the Validation Service */
+    @SerializedName("hostname") val hostname: String,
+
+    /** The SHA-256 fingerprint of the certificate of the Validation Service */
+    @SerializedName("fingerprint256") val fingerprint256: ByteString
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/allowlist/data/DccTicketingValidationServiceAllowListEntry.kt:17]: @SerializedName can be removed now ?

**Review by mtwalli — COMMENTED** [review]
LGTM, I can't approve ,since I'm author

**Review by jurajkusnier — APPROVED** [review]
LGTM

**Pull request merged**